libraries/Microsoft.Bot.Builder.ApplicationInsights/BotTelemetryClient.cs

```diff
@@ -14,7 +14,7 @@ public class BotTelemetryClient : IBotTelemetryClient
 
         public BotTelemetryClient(TelemetryClient telemetryClient)
         {
-            _telemetryClient = telemetryClient;
+            _telemetryClient = telemetryClient ?? throw new ArgumentNullException(nameof(telemetryClient));
```

Comment:
Realized you didn't have a test for this scenario and when the test went red I realized the code didn't even handle the scenario so I fixed it. Hurray for unit tests!